Scripts/Utilities/ObjectUtils.cs

```diff
@@ -61,7 +62,7 @@ public static bool IsInLayer(GameObject o, string s)
 		public static GameObject CreateEmptyGameObject(String name = null, Transform parent = null)
 		{
 			GameObject empty = null;
-			if (string.IsNullOrEmpty(name))
+			if (String.IsNullOrEmpty(name))
```

Comment:
Why this change? Also why String above and not string?

Come to think of it, this method is a little odd to me. Outside of EVR it just returns null?